doc/profiles.rst

```diff
@@ -61,3 +61,4 @@ Profile Summary
 #. refkit-image-common: A configuration with interactive tools but without any special software in it. Recommended target if you are not interested in any given profile.
 #. refkit-image-computervision: A profile with tools and configuration for computer vision use cases.
 #. refkit-image-gateway: A profile with tools and configuration for acting as an IoT sensor.
+#. refkit-image-industrial: A profile with core ROS Industrial packages intended to be used as a base for industrial robotics use cases.
```

Comment:
This is just a thought, but should the image (and profile) rather be called `industrial-robotics`?